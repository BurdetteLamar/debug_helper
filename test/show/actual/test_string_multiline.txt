---
String (message='test_string_multiline' size=19 encoding=UTF-8 ascii_only=true bytesize=19):
- |
  foobar
  snafu
  janfu
