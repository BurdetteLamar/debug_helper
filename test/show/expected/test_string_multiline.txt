---
String (size=19 encoding=UTF-8 name=test_string_multiline):
- |
  foobar
  snafu
  janfu
